randovania/games/prime3/logic_database/Bryyo - Fire.txt

```diff
@@ -140,6 +140,8 @@ Extra - asset_id: 4215957264467390882
               Any of the following:
                   After Imperial Hall Grapple Lasso Walls or Bomb/Spring Space Jump (Advanced)
                   Boost Ball and Bomb/Spring Space Jump (Intermediate)
+                  # https://www.youtube.com/watch?v=Z-XICpMJWoE
```

Comment:
for the sake of consistency, we like to have all yt video links truncated so that they look like
`https://youtu.be/[videoid]` rather than `https://youtube.com/watch?v=[videoid]`
Small change I know but would be greatly appreciated.

Reply:
If you could also go and leave comments describing the trick you perform next to the video links where applicable would also be very helpful, doesn't have to be long.

Reply:
Honestly, we really should have this done automatically by rdv.